cafebabel/templates/articles/macros.html

```diff
@@ -6,9 +6,9 @@
   {% endif %}
 {% endmacro %}
 
-{% macro profile_image(author) %}
+{% macro profile_image(author, dimensions='200x200') %}
```

Comment:
Right. What about Retina screens?

Reply:
The state of the art is this: https://timkadlec.com/remembers/2018-03-22-compressive-images-revisited/ it is possible to do that with this implementation but it will not be in that PR.